include/IEntity.hpp

```diff
@@ -22,14 +22,14 @@ namespace arcade
     public:
         virtual ~IEntity() = default;
 
-        virtual std::vector<std::unique_ptr<IComponent>> &getComponents() = 0;
-        // TODO: vérifier la faisibilité des tags avec juste cette méthode dans
-        // l'interface
-        virtual bool hasTag(const std::string &tag) = 0;
+        virtual std::vector<std::unique_ptr<IComponent>> &getComponents() = 0; /* > returning the entity all components as a vector of IComponents */
```

Comment:
Maybe reformulate to:

Return all the components of an entity, as a vector of IComponents